docs/src/pages/guides/migrating-to-react-query-3.md

```diff
@@ -116,6 +116,46 @@ Together, these provide the same experience as before, but with added control to
 
 **Notice that it's now a function instead of a property**
 
+### `QueryStatus` has been changed from an [enum](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) to a [union type](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#union-types)
```

Comment:
Can you please mention somewhere that this applies to `TypeScript` only - maybe a dedicated _TypeScript_ sub-section?

Reply:
yup :+1:

Reply:
done!